### ISSUE TITLE
Привет есть ли возможность создать патч на cmp 40 для полноценной работы ?

### ISSUE THREAD

**Issue by KIRILL-35** [dartraiden/NVIDIA-patcher#42]

**Comment by dartraiden** [comment]
А что неполноценно сейчас?

- шина: напайкой недостающих элементов увеличивается с x4 до x16
- версия шины: заблокирована аппаратно на 1.1, программно ничего сделать нельзя
- TDP: есть какие-то жалобы на низкое потребление, но пока мало статистики, касается ли это всех CMP-карт или нет
- NVENC: вроде, поддерживается, возможно, следует использовать модифицированные файлы с приставкой NVENC-

**Comment by KIRILL-35** [comment]
Она на монете cfx conflux при загрузки дак файла скидывает напряжения до 80 W На других монетах всё норм . 
Я и подумал может есть что то для этого ?

**Comment by KIRILL-35** [comment]
А пропайка может что то изменить ?

**Comment by dartraiden** [comment]
Будет шина x16 вместо x4, в играх это самое то.

**Comment by dartraiden** [comment]
Единственное, что  мне приходит в голову, попробуй через диспетчер устройств руками подсунуть драйвер и вручную выбрать RTX 2070.

**Comment by KIRILL-35** [comment]
Установил в пк диспетчере устройств показывает 2070супер но галочки так и не появились

**Comment by dartraiden** [comment]
С зачисткой предыдущего через DDU и при отключённом доступе в сеть?

**Comment by operador777** [comment]
> I installed it in the device manager on my PC and it shows 2070super but the checkboxes still haven’t appeared.

I also have the CMP 40 HX, and I performed the manual driver installation as Dartraiden suggests, and it's working correctly for me. Make sure to disconnect the internet, uninstall the drivers using DDU, restart, and proceed with the manual installation.

![image](https://github.com/dartraiden/NVIDIA-patcher/assets/19232315/10e28d2e-fffd-4c22-98d9-d3fcd54f220f)

**Comment by operador777** [comment]
Apart from that, you can follow the detailed instructions in the Readme regarding [Shadowplay](https://github.com/dartraiden/NVIDIA-patcher/#shadowplay-also-known-as-the-nvidia-overlay-on-cards-that-support-nvenc). Once activated, you'll navigate to the performance tab and can slightly increase the power limits. I'm not sure if it has any benefits for mining, but it does provide an improvement for gaming.

![image](https://github.com/dartraiden/NVIDIA-patcher/assets/19232315/b3aad7a0-926d-4d90-b784-1c05701ed707)

**Comment by dartraiden** [comment]
> disconnect the internet

Yes, it's important step. After removing the driver, Windows starts downloading and installing the driver automatically. Therefore, either turn off the internet to prevent Windows from doing this. Or wait until Windows installs it own driver and only then install the modified driver (with "clean installation" option).
